Source/Core/Core/PowerPC/JitArm64/JitArm64_Branch.cpp

```diff
@@ -125,6 +134,8 @@ void JitArm64::bcx(UGeckoInstruction inst)
   JITDISABLE(bJITBranchOff);
 
   ARM64Reg WA = gpr.GetReg();
+  ARM64Reg tmp_reg = inst.LK ? gpr.GetReg() : WA;
```

Comment:
Just a nitpick, but the code would look nicer if you renamed `tmp_reg` to WB and unlocked it at the same spot as we unlock WA.